config/sync/user.role.os2loop_user_administrator.yml

```diff
@@ -82,6 +83,7 @@ permissions:
   - 'create os2loop_post content'
   - 'create os2loop_question content'
   - 'create os2loop_section_page content'
+  - 'create terms in os2loop_media_library'
```

Comment:
This looks like stuff from https://github.com/itk-dev/os2loop/pull/161 and should be removed from here.